buildstockbatch/gcp/gcp.py

```diff
@@ -192,13 +195,38 @@ def validate_gcp_args(project_file):
                 f"Artifact Registry repository {repo} does not exist in project {gcp_project} and region {region}"
             )
 
+        # Check post-processing resources
+        pp_env = cfg["gcp"].get("postprocessing_environment")
+        if pp_env:
+            cpus = pp_env.get("cpus", GcpBatch.DEFAULT_PP_CPUS)
+            memory = pp_env.get("memory_mib", GcpBatch.DEFAULT_PP_MEMORY_MIB)
+
+            # Allowable values are documented at:
+            # https://cloud.google.com/python/docs/reference/run/latest/google.cloud.run_v2.types.ResourceRequirements
+            # https://cloud.google.com/run/docs/configuring/services/cpu
+            # https://cloud.google.com/run/docs/configuring/services/memory-limits
+            cpus_to_memory_limits = {
+                1: (512, 4096),
+                2: (512, 8192),
+                4: (2048, 16384),
+                8: (4096, 327681),
+            }
+
+            assert cpus in cpus_to_memory_limits, "gcp.postprocessing_environment.cpus must be 1, 2, 4 or 8"
+            min_memory, max_memory = cpus_to_memory_limits[cpus]
+            assert min_memory <= memory, (
+                f"gcp.postprocessing_environment.memory_mib must be at least {min_memory} for {cpus} CPUs. "
```

Comment:
provide them with the link to more info (...google.cloud.run_v2.types.ResourceRequirements) here and in the next assert?